packages/scaffold-ui/src/partials/w3m-connect-external-widget/index.ts

```diff
@@ -29,8 +29,11 @@ export class W3mConnectExternalWidget extends LitElement {
   // -- Render -------------------------------------------- //
   public override render() {
     const externalConnectors = this.connectors.filter(connector => connector.type === 'EXTERNAL')
+    const filteredOutCoinbaseConnectors = externalConnectors.filter(
+      connector => connector.id !== 'coinbaseWalletSDK'
+    )
```

Comment:
have you tried what would happen if it were to be one of the first results of the list / featured? 

In that case its valid to be on first screen

I think it would have this same ID and get filtered but not completely sure

Reply:
+different E2E test variations of `featuredWalletIds` use cases, would be really beneficial

Reply:
It wouldn't be filtered out if it is added with the `featuredWalletIds` list, because those wallets will be rendered in the `connect-featured-widget`, but I will add some tests